### PR TITLE
feature/added json-download

### PR DESCRIPTION
**Pull request by gecBurton**
## Context
We want to be able to see useful data quickly, particularly for tracking token spending.

## Changes proposed in this pull request
Adds a new 'download json' option under the User model in Admin, which - thanks to the magic ⚡ of Django Rest Framework Serialisers 🧙 - allows us to see Chats, ChatMessages, token and File usage.

## Guidance to review
To try this locally, go to Django Admin > Users . Click on the select box to select all Users, and choose 'Export Selected' from the dropdown menu. _Does the json look reasonable?_

(Note: this is a different flow from clicking the 'Export' button, which is based on the User model alone and is a default export implementation that allows us to get quick, uncustomised, data)

## Things to check

- [ ] I have added any new ENV vars in all deployed environments
- [x] I have tested any code added or changed
- [ ] I have run integration tests

### CODE REVIEW

File: django_app/redbox_app/redbox_core/admin.py [hunk 1]
```diff
@@ -12,12 +13,23 @@
...
 
 logger = logging.getLogger(__name__)
 core_api = CoreApiClient(host=settings.CORE_API_HOST, port=settings.CORE_API_PORT)
 
 
 class UserAdmin(ImportExportMixin, admin.ModelAdmin):
+    def export_as_json(self, request, queryset: QuerySet):  # noqa:ARG002
+        user_data = UserSerializer(many=True).to_representation(queryset)
+        response = HttpResponse(json.dumps(user_data), content_type="text/json")
+        response["Content-Disposition"] = "attachment; filename=chat.csv"
```
Comment (lmwilkigov) [inline django_app/redbox_app/redbox_core/admin.py:26]: should the chat.csv be chat.json?

Comment (gecBurton) [reply django_app/redbox_app/redbox_core/admin.py:26]: yes 100%! thanks you

File: django_app/redbox_app/redbox_core/serializers.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        model = Chat
+        fields = ("name", "messages")
+
+
+class UserSerializer(serializers.ModelSerializer):
+    chats = ChatSerializer(many=True, read_only=True)
+
+    class Meta:
+        model = User
+        fields = ("is_staff", "business_unit", "grade", "email", "ai_experience", "profession", "chats")
```
Comment (gecBurton) [inline django_app/redbox_app/redbox_core/serializers.py:45]:
```suggestion
        fields = ("is_staff", "business_unit", "grade", "email", "ai_experience", "profession", "chats", "is_developer")
```

File: django_app/redbox_app/redbox_core/admin.py (hunk 1 above)
Comment (gecBurton) [inline django_app/redbox_app/redbox_core/admin.py:26]:
```suggestion
        response["Content-Disposition"] = "attachment; filename=data-export.json"
```

**Review by rachaelcodes — APPROVED** [review]

**Comment by rachaelcodes** [comment]
Approved in conversation with @gecBurton

**Pull request merged**